packages/components/src/internal/components/search/EntityFieldFilterModal.tsx

```diff
@@ -179,9 +190,13 @@ export const EntityFieldFilterModal: FC<Props> = memo(props => {
             queryLabels[parentValue] = label;
         });
         const filterErrors = getFieldFiltersValidationResult(validDataTypeFilters, queryLabels);
-        if (!filterErrors) onFind(entityDataType.instanceSchemaName, validDataTypeFilters);
-        else setFilterError(filterErrors);
-    }, [onFind, validDataTypeFilters]);
+        if (!filterErrors) {
+            if (metricFeatureArea) api.query.incrementClientSideMetricCount(metricFeatureArea, 'filterModalApply');
```

Comment:
The check for `metricFeatureArea` is not needed as `incrementClientSideMetricCount` checks for that.

Reply:
good call. done